core/src/main/java/com/radixdlt/consensus/Vertex.java

```diff
@@ -260,17 +260,17 @@ public HashCode getParentVertexId() {
   public long getEpoch() {
     var epoch = getParentHeader().getLedgerHeader().getEpoch();
     // If vertex is genesis, the parent will point to the previous epoch so must add 1
-    return round.isGenesis() ? epoch + 1 : epoch;
+    return round.isEpochInitial() ? epoch + 1 : epoch;
   }
 
   public BFTHeader getGrandParentHeader() {
     return getQCToParent().getParentHeader();
   }
 
-  public boolean touchesGenesis() {
-    return this.getRound().isGenesis()
-        || this.getParentHeader().getRound().isGenesis()
-        || this.getGrandParentHeader().getRound().isGenesis();
+  public boolean touchesEpochInitialRound() {
```

Comment:
Can we do this renaming in another PR? This PR has gotten too big I think and having trouble finding the actual logic updates vs renames.